contracts/.solhint.json

```diff
@@ -9,6 +9,7 @@
     "compiler-version": ["off", "^0.4.24"],
     "avoid-throw": false,
     "var-name-mixedcase": false,
-    "mark-callable-contracts": false
+    "mark-callable-contracts": false,
+    "func-visibility": ["warn", { "ignoreConstructors": true }]
```

Comment:
Note this only applies to >=0.7, however <0.7 will not compile without constructor visibility so we are ok